Interface/AddOns/nMainbar/Libs/LibActionButton-1.0-ElvUI/LibActionButton-1.0.lua

```diff
@@ -102,14 +102,14 @@ local Custom = setmetatable({}, {__index = Generic})
 local Custom_MT = {__index = Custom}
 
 local type_meta_map = {
-	empty  = Generic_MT,
-	action = Action_MT,
-	--pet    = PetAction_MT,
-	spell  = Spell_MT,
-	item   = Item_MT,
-	macro  = Macro_MT,
-	toy    = Toy_MT,
-	custom = Custom_MT
+    empty  = Generic_MT,
+    action = Action_MT,
+    --pet    = PetAction_MT,
+    spell  = Spell_MT,
+    item   = Item_MT,
+    macro  = Macro_MT,
+    toy    = Toy_MT,
+    custom = Custom_MT
```

Comment:
Not sure it's a good idea to change the indentation of a library like this (same with quotes). It will just make it harder to perform diffs against the original. Which is useful to figure out what changes we've made, but also to see what changes _they've_ made, e.g. bug fixes.

Reply:
Ok I restored the old formatting and updated to the latest version of the lib so it should be good now.